phoneApp/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>phoneApp</title>
+      <meta charset="utf-8" />
+      <link href="https://stackpath.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+      <link rel="stylesheet" type="text/css" href="style.css">
+   </head>
+   <body>
+
+      <script src="constants.js" defer></script>
+		<script src="headerAndFooter.js" defer></script>
```

Comment:
if you set attribute "defer" for every script you can't be sure your scripts loaded synchronously.

For example, your script "index" can be loaded before **HeaderAndFooter** and fatal error would happen.

So better build application with "predictable" flow